backend/src/providers/transactions.ts

```diff
@@ -273,12 +404,12 @@ export const mapRpcActionToAction = (action: RPC.ActionView): Action => {
 type ReceiptExecutionOutcome = {
   tokens_burnt: string;
   logs: string[];
-  outgoing_receipts?: NestedReceiptWithOutcome[];
+  outgoing_receipts?: NestedReceiptWithOutcomeOld[];
```

Comment:
I was trying to refactor types and methods to be similar and it's almost identical. But I can't replace the "old" methods and types with the "new" one because I need to make changes with mock data of our tests (and it's not a part of this PR). That's why I added `Old` suffix to methods and types (couple of them actually)